vnext/ReactUWP/Views/FrameworkElementViewManager.cpp

```diff
@@ -95,6 +96,38 @@ void FrameworkElementViewManager::UpdateProperties(ShadowNodeBase* nodeToUpdate,
           continue;
         }
       }
+      if (propertyName == "transform")
```

Comment:
Does the property need to be added in TransferProperties() as well?

Reply:
good catch, fixed